src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const STATUS = 'bookstore/categories/status';
```

Comment:
- [ ] [REQIRED]  Kindly refactor the value of your Action Type to use the [ducks](https://github.com/erikras/ducks-modular-redux) pattern: `const STATUS = bookstore/books/STATUS`

Reply:
Hi @Whoistolu i think this one in categories folder I think it is correct.

Reply:
Yes, please.